### PR TITLE
Renjani2022

### PR DESCRIPTION
**Pull request by renjani2022**
### General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [ ] PR has a descriptive title
- [ ] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [ ] reviewers are assigned
- [ ] the PR contributes only one focused change
- [ ] It is in the appropriate column in the project board (if necessary)
- [ ] has short and clear description
- [ ] is linked to an issue (if it is related)
- [ ] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work
- [x] any embedded code snippets are formatted

### CODE REVIEW

File: student-bios/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (yildiztugba) [inline student-bios/package.json:1]: please delete this file

File: student-bios/Renjani2022.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Introduction
+Hi, I'm Renjani!
```
Comment (yildiztugba) [inline student-bios/Renjani2022.md:2]: you have a format error: 

> Checking formatting...
[warn] student-bios/Renjani2022.md
[warn] Code style issues found in the above file. Forgot to run Prettier?
Error: Process completed with exit code 1.

Please run the scripts in the terminal one by one 
npm run format
npm run format:check
npm run lint:ls
npm run lint:md
npm run spell-check

**Review by yildiztugba — CHANGES_REQUESTED** [review]

File: student-bios/Renjani2022.md (hunk 2 above)
Comment (renjani2022) [reply student-bios/Renjani2022.md:2]: Thanks @yildiztugba. I have done now. And what shall I do next? Shall I apply for the new pull request? Thanks a lot

Comment (yildiztugba) [reply student-bios/Renjani2022.md:2]: i made 2 comments, please look, you have still format error. dont forget to run script and push again

File: student-bios/package.json (hunk 1 above)
Comment (yildiztugba) [reply student-bios/package.json:1]: @renjani2022 here my comment

**Comment by yildiztugba** [comment]
congrats! :tada: you have no error. You need to go through the checklist. I noticed two things: 1. the name of your file has capital letter (R). you need to change with lowercase 'r'. and then again push changes. 2. you need to link your pr with the issue related to this task. (on the right side, click development part and find your issue)

**Comment by renjani2022** [comment]
> 

Thanks much @yildiztugba. I have renamed my file and tried to push as you suggested, it did work in VSC, but when I checked my branch, there is still no change in the github. Do you have any suggestion what's wrong with it? Thanks a lot for you help

**Comment by renjani2022** [comment]
> #89 in this pr, john has a solution for that. you can change the name of your file on Github manually. go to the link: https://github.com/HYF-Class19/home/tree/My_New_Branch/student-bios find your md file and edit&change the name.

**Comment by renjani2022** [comment]
Hi @yildiztugba, I think I have made a fatal step. By accident I press closed for this pull request, can I activate/open it again?

**Pull request closed without merging**